cs/src/Contracts/TunnelStatus.cs

```diff
@@ -94,4 +94,10 @@ public class TunnelStatus
     /// </summary>
     [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
     public RateStatus? ApiUpdateRate { get; set; }
+
+    /// <summary>
+    /// Gets or sets the current value and limit for the rate of http requests to the tunnel.
```

Comment:
Should we mention "web forwarding" so it's clear this isn't management plane operations on the tunnel? Same for the other language comments.

Reply:
good point, added

Reply:
Eventually I'd like this to include HTTP requests through desktop / SDK client forwarding also. But that will require  changing the protocol, as I've mentioned before.